src/main/scala/is/hail/io/vcf/LoadVCF.scala

```diff
@@ -9,14 +9,187 @@ import is.hail.utils._
 import is.hail.variant._
 import org.apache.hadoop
 import org.apache.hadoop.conf.Configuration
+import org.apache.spark.rdd.RDD
 
+import scala.annotation.switch
 import scala.collection.JavaConversions._
 import scala.language.implicitConversions
 import scala.collection.mutable
 import scala.io.Source
 
 case class VCFHeaderInfo(sampleIds: Array[String], infoSignature: TStruct, vaSignature: TStruct, genotypeSignature: TStruct, canonicalFlags: Int)
 
+object VCFLine {
+  def numericValue(c: Char): Int = {
+    if (c < '0' || c > '9')
+      fatal(s"invalid character '$c' in integer literal")
```

Comment:
these errors will be very hard to localize and interpret. Ideally we can use `pos` to print a carat within the string, or print an error about which field was the problem.

Reply:
Yeah, for a big file without the caret it's useless.  I was thinking of throwing an exception with the location and then pulling 60 chars of context out.  I'll do it tomorrow.